libshaderc/include/shaderc.h

```diff
@@ -113,7 +113,7 @@ void shaderc_compile_options_release(shaderc_compile_options_t options);
 void shaderc_compile_options_add_macro_definition(
     shaderc_compile_options_t options, const char* name, const char* value);
 
-// Set the compiler to emit a disassembly text instead of a binary. In
```

Comment:
I rebased my local branch to google's master branch and the master branch does the changes. But the remote branch stay unchanged, so this diff also shows some changes in master.